resources/plan.md

```diff
@@ -196,25 +202,56 @@ Rhombus project, will make the calls at this phase.
 Failure is no longer an option at this point for the new language,
 but its prominence within Racket will depend on the community.
 
+CHANGE POLICY: Breaking changes are allowed, but rare. Every breaking
+change needs a clearly identified transition path for all Rhombus
+users.
+
 CONCLUSION: The new language is at least as well supported and 
 available as `#lang racket`. 
 
+Phase 5: Production (eons)
+-------------------
+
+GOAL AND OUTPUT: Long-term maintenance and development of Rhombus.
+
+PROCESS: No programming language design is ever final, but this phase
+reflects the point where Rhombus is best characterized as in compete
+and ready for production. The language is maintained and managed by
+the Rhombus and Racket community.
+
+CHANGE POLICY: Breaking changes are generally disallowed. Although
+*any* change is a breaking change to some potential program
+(otherwise, it wouldn't be a change), "breaking change" at this point
+means breaking a package that is publically registered in place such
+as [https://pkgs.racket-lang.org](https://pkgs.racket-lang.org).
+In cooperation with the broader Rhombus community, Rhombus management
+may decide to accept a breaking change, but only as a carefully
+reasoned exemption to the general prohibition against breaking changes.
+
 
 What Will Happen to Racket by the End 
 ------------------------------------- 
 
-Transitioning does not mean that Racket will disappear. 
+Reaching the Production phase of Rhombus does not mean that Racket will disappear.
 
-- Existing `#lang racket` programs will continue to run beyond Phase 4. 
+- Existing `#lang racket` programs will continue to run.
 
-- The documentation for `#lang racket` will co-exist with whatever we 
-call the new language. 
+- The documentation for `#lang racket` will co-exist with whatever we
+call the new language.
```

Comment:
We've decided on a name, so we can just use "Rhombus" here now.

Reply:
Good point!